Dockerfile

```diff
@@ -0,0 +1,30 @@
+FROM alpine:3.6
+
+ADD . /app
+
+RUN cd app && sh docker/container-install.sh
+
+WORKDIR /app
+
+ENV MYSQL_HOST localhost
```

Comment:
Doesn't it kind of defeat the purpose of docker to run a full stack inside one container?  The [MariaDB Docker image](https://hub.docker.com/_/mariadb/), for example, gives an example of how to link a database container with an application container.  There is also the case of connecting to an external database (e.g. AWS RDS, or a self-hosted db server), as I do today (3 app servers connecting to 1 db server).